lib/broadside/command.rb

```diff
@@ -0,0 +1,96 @@
+require 'pp'
+require 'tty-table'
+
+module Broadside
+  module Command
+    class << self
+      def bootstrap(options)
+
+        EcsDeploy.new(options[:target], options).bootstrap
```

Comment:
maybe do `EcsDeploy.new(options.delete(:target), options) so there's not an extra key in there later (can be handy for validation to strip out the keys as you consume them; see what i did in the `Target` validation)